lib/liquid/tags/echo.rb

```diff
@@ -20,6 +20,20 @@ def initialize(tag_name, markup, parse_context)
     def render(context)
       @variable.render_to_output_buffer(context, +'')
     end
+
+    def name
+      @variable.name
+    end
+
+    def filters
+      @variable.filters
+    end
+
+    class ParseTreeVisitor < Liquid::ParseTreeVisitor
+      def children
+        [@node.name] + @node.filters.flatten
```

Comment:
This would be better if it was a reference to the Variable's ParseTreeVisitor.children instead. But I'm a ruby noob and I don't know how to do that.